src/examples/libpmem2/unsafe_shutdowns.c

```diff
@@ -0,0 +1,404 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
I suggest to put it in separate dir (look at libpmemobj examples)
src/examples/libpmem2/**unsafe_shutdowns**/unsafe_shutdowns.c